x-pack/filebeat/input/entityanalytics/provider/activedirectory/activedirectory.go

```diff
@@ -345,7 +345,7 @@ func (p *adInput) doFetchUsers(ctx context.Context, state *stateStore, fullSync
 		since = state.whenChanged
 	}
 
-	entries, err := activedirectory.GetDetails(p.cfg.URL, p.cfg.User, p.cfg.Password, p.baseDN, since, p.cfg.UserAttrs, p.cfg.GrpAttrs, p.cfg.PagingSize, nil, p.tlsConfig)
+	entries, err := activedirectory.GetDetails("(&(objectCategory=person)(objectClass=user))", p.cfg.URL, p.cfg.User, p.cfg.Password, p.baseDN, since, p.cfg.UserAttrs, p.cfg.GrpAttrs, p.cfg.PagingSize, nil, p.tlsConfig)
```

Comment:
would it make sense to make these query values dynamic rather than hardcoded if by chance something changes and these categories are somehow changed ?